source/model.js

```diff
@@ -91,57 +107,61 @@ function ComputerPlayer(name, symbol, field){
 ComputerPlayer.prototype = Object.create(Player.prototype);
 ComputerPlayer.prototype.constructor = ComputerPlayer;
 
-function Game(humanPlayer, firstPlayerName, secondPlayerName){
+function Iterator(array) {
+    let index = 0;
+    this.getCurrent = function(){
```

Comment:
Please keep object members visually separated from each other. One empty line between them is usually enough.